config.json

```diff
@@ -1,9 +1,9 @@
 {
-  "baseConfig": "trifid:config-fs.json",
+  "baseConfig": "trifid:config-fetch.json",
   "handler": {
     "root": {
       "options": {
-        "path": "node_modules/tbbt-ld"
+        "url": "file:./node_modules/tbbt-ld"
```

Comment:
I think the URL should be `file:./node_modules/tbbt-ld/dist/tbbt.nq`.

When the config is working, it should be possible to open the URLs from the examples like: http://localhost:8080/data/person/amy-farrah-fowler

Reply:
Ah, no media type on the file system... Can be fixed with the `contentType` property and  `application/n-quads` value. The N-Quad should be used, otherwise the split is required.

Reply:
Nice, this works!